docusaurus.config.js

```diff
@@ -17,77 +17,72 @@ const config = {
     [
       require.resolve('@docusaurus/plugin-client-redirects'),
       {
-        redirects: [
-        ]
-      }
-    ]
+        redirects: [],
```

Comment:
you have somekind of prettier / lint enabled ? Why does it add commas here i dont think this is a needed update.